packages/genesys-spark-components/src/components/stable/gux-tag/example.html

```diff
@@ -36,6 +36,42 @@ <h2>Disabled</h2>
   <gux-tag disabled accent="invalid">Invalid</gux-tag>
 </div>
 
+<h2>Size</h2>
+<div class="tags">
+  <gux-tag size="small">Small</gux-tag>
+  <gux-tag size="large">Large</gux-tag>
+</div>
+
+<h2>Emphasis</h2>
+<div class="tags">
+  <div>
+    <gux-tag emphasis="subtle" accent="1">Subtle 1</gux-tag>
```

Comment:
Would `variant` be a more flexible property name?

Reply:
@daragh-king-genesys Variant doesn't sit well with UX, It has a particular meaning to designers in Figma & the feel 'emphasis' has more meaning in this case.

Badge has the opposite default value so we can't use a 'bold' prop however they feel they could need a 3rd option in the future & don't want to use a boolean.